.github/scripts/check-template-and-add-labels.ts

```diff
@@ -215,6 +207,25 @@ function extractTemplateTypeFromBody(body: string): TemplateType {
   return TemplateType.None;
 }
 
+// This helper function extracts regression stage (Development, Testing, Production) from bug report issue's body.
+function extractRegressionStageFromBugReportIssueBody(
+  body: string,
+): RegressionStage | undefined {
+  const detectionStageRegex = /### Detection stage\s*\n\s*(.*)/i;
+  const match = body.match(detectionStageRegex);
+  const extractedAnswer = match ? match[1].trim() : undefined;
+
+  if (extractedAnswer === 'On the development branch') {
+    return RegressionStage.Development;
+  } else if (extractedAnswer === 'During release testing') {
+    return RegressionStage.Testing;
+  } else if (extractedAnswer === 'In production') {
+    return RegressionStage.Production;
+  } else {
+    return undefined;
+  }
+}
```

Comment:
Nit: using switch can make the condition easier to read and manage
```suggestion
switch(extractedAnswer) {
  case('On the development branch'):
    return RegressionStage.Development;
  case ('During release testing'):
    return RegressionStage.Testing;
  case ('In production'):
    return RegressionStage.Production;
    default: 
    return undefined;
}
```

Reply:
Sure, it's now fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/25691/commits/0675b65ab5dea44f83c0c5ed785e62432e3306b5)